doc/source/serve/configuring-serve-deployments.md

```diff
@@ -176,11 +179,46 @@ For example, if you're using OpenCV, you'll need to manually set the number of t
 You can check the configuration using `cv2.getNumThreads()` and `cv2.getNumberOfCPUs()`.
 :::
 
+(managing-deployments-handling-dependencies)=
+
+## Handling Dependencies
+
+Ray Serve supports serving deployments with different (possibly conflicting)
+Python dependencies.  For example, you can simultaneously serve one deployment
+that uses legacy Tensorflow 1 and another that uses Tensorflow 2.
+
+This is supported on Mac OS and Linux using Ray's {ref}`runtime-environments` feature.
+As with all other Ray actor options, pass the runtime environment in via `ray_actor_options` in
+your deployment.  Be sure to first run `pip install "ray[default]"` to ensure the
+Runtime Environments feature is installed.
+
+Example:
+
+```{literalinclude} ../../../python/ray/serve/examples/doc/conda_env.py
+```
+
+:::{tip}
+Avoid dynamically installing packages that install from source: these can be slow and
```

Comment:
It is not very clear what dynamically installing packages means here. From the conda_env.py example, does it count as dynamically installing packages ? :)

Reply:
Ah good point, let me improve this

Reply:
Maybe let's merge this and I'll make the improvement in a followup edit pass.  This PR is just to reorganize the doc structure

Reply:
Addressed it in this PR, it should be clearer now.